### PR TITLE
DVX-5064: Isolate CreateDBInstanceInput comparison

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

**Comment by miguelbernadi** [comment]
+1

File: mcc/odin/odin/odin_test.go [hunk 1]
```diff
@@ -410,6 +410,26 @@ var getCreateDBInstanceInputCases = []getCreateDBInstanceInputCase{
 	},
 }
 
+func equalsCreateDBInstanceInput(input1, input2 *rds.CreateDBInstanceInput) (result bool) {
+	result = true
+	if *input1.AllocatedStorage != *input2.AllocatedStorage {
+		result = false
```
Comment (christofdamian) [inline mcc/odin/odin/odin_test.go]: go confused me with the result and empty return.
you could also set ```result = false``` first and then return early in each if and set ```result = true``` in the last line.

**Comment by christofdamian** [comment]
+1

**Comment by ifosch** [comment]
Followed @christofdamian approach ;)

**Comment by christofdamian** [comment]
much faster now ;-)

**Comment by ifosch** [comment]
Followed @miguelbernadi suggestion on the `if` replaced with `switch`: Less lines, more readable :)

**Pull request merged**